### DISCUSSION TITLE
Can GNPy be able to simulate power balancing of C+L Band since they are prone to larger SRS tilts ?

### DISCUSSION THREAD

**Discussion by ssai94** [Telecominfraproject/oopt-gnpy#445] [Q&A] [answered]
Dear Contributors,

I have a question on the capability of GNPy, if it simulates channel power balancing across C+L Band in a given network topology ?
If it does, can anyone please let me know the location of this part of the code ?

Since SRS tilts can kick in here in C+L scenario very well due to the larger optical bandwidth and powers being used in the fiber, want to know if GNPy has already used a power balancing technique for this, which can optimize the performance of per-channel OSNRs.

Thanks !

**Comment by jktjkt** [comment, marked as answer]
Hi, the code currently lets all the `ROADM` nodes equalize all channels. This has worked well so far for homogeneous signal loads, but it will be changed as we merge the [`mixed-rate` patches](https://review.gerrithub.io/q/topic:mixed-rate+project:Telecominfraproject/oopt-gnpy) where the code will have to consider, e.g., PSD.
